glyphs2fontir/src/source.rs

```diff
@@ -731,15 +731,38 @@ impl Work<Context, WorkId, WorkError> for GlyphIrWork {
 
         // Glyphs have layers that match up with masters, and masters have locations
         let mut axis_positions: HashMap<Tag, HashSet<NormalizedCoord>> = HashMap::new();
+        let axes_by_name = font_info.axes.iter().map(|a| (a.tag, a)).collect();
         for instance in glyph.layers.iter() {
-            let Some(master_idx) = font_info.master_indices.get(instance.layer_id.as_str()) else {
+            // skip not-yet-supported types of layers (e.g. alternate, color, etc.)
+            if !(instance.is_master() || instance.is_intermediate()) {
```

Comment:
Optional, if the bools here are exclusive - you can be a master, you can be an intermediate, you can be neither, but you cannot be both - then I wonder if we should have something akin to enum LayerType.

Reply:
> if the bools here are exclusive 

while that is true for master vs non-master layer types, however non-master layer types can combine with one another, e.g you _can_ have an intermediate, alternate, color layer in fact.. though it bogs my mind